dascore/io/mseed/utils.py

```
@@ -0,0 +1,667 @@
+    if not _is_seed_code(header[8:13]):
+        return False
+    if not _is_seed_code(header[15:18], allow_space=False):
+        return False
+    if not _is_seed_code(header[18:20]):
+        return False
+    try:
+        year, day = unpack(">HH", header[20:24])
+        hour, minute, second = header[24:27]
+        _unused, frac_seconds, sample_count = unpack(">BHH", header[27:32])
+        data_offset, blockette_offset = unpack(">HH", header[44:48])
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)